src/utils/config.ts

```diff
@@ -50,19 +55,36 @@ export const SpidParams = t.intersection([
 
 export type SpidParams = t.TypeOf<typeof SpidParams>;
 
+const JWTParams = t.union([
+  t.interface({
+    ENABLE_JWT: t.literal(true),
+    JWT_TOKEN_EXPIRATION: NonNegativeInteger,
+    JWT_TOKEN_ISSUER: NonEmptyString,
+    JWT_TOKEN_PRIVATE_RSA_KEY: NonEmptyString
+  }),
+  t.interface({
+    ENABLE_JWT: t.literal(false)
+  })
+]);
+type JWTParams = t.TypeOf<typeof JWTParams>;
+
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.intersection([
   t.interface({
     isProduction: t.boolean
   }),
   RedisParams,
-  SpidParams
+  SpidParams,
+  JWTParams
 ]);
 
 // No need to re-evaluate this object for each call
 const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   ...process.env,
+  ENABLE_JWT: fromNullable(process.env.ENABLE_JWT)
+    .map(_ => _.toLowerCase() === "true")
+    .getOrElseL(() => false),
```

Comment:
We still need the decoding for `JWT_TOKEN_EXPIRATION`
```suggestion
  ENABLE_JWT: fromNullable(process.env.ENABLE_JWT)
    .map(_ => _.toLowerCase() === "true")
    .getOrElseL(() => false),
  // -1 value fail IConfig decode if ENABLE_JWT is true
  JWT_TOKEN_EXPIRATION: fromNullableE(-1)(process.env.JWT_TOKEN_EXPIRATION)
    .chain(_ => IntegerFromString.decode(_).mapLeft(() => -1))
    .fold(identity, identity),
```